src/table.ts

```diff
@@ -1114,53 +1116,8 @@ class Table extends ServiceObject {
    * @param {string} [metadata.jobId] Custom job id.
    * @param {string} [metadata.jobPrefix] Prefix to apply to the job id.
    * @returns {WritableStream}
-   *
-   * @throws {Error} If source format isn't recognized.
-   *
-   * @example
-   * const BigQuery = require('@google-cloud/bigquery');
-   * const bigquery = new BigQuery();
-   * const dataset = bigquery.dataset('my-dataset');
-   * const table = dataset.table('my-table');
-   *
-   * //-
-   * // Load data from a CSV file.
-   * //-
-   * const request = require('request');
-   *
-   * const csvUrl = 'http://goo.gl/kSE7z6';
-   *
-   * const metadata = {
-   *   allowJaggedRows: true,
-   *   skipLeadingRows: 1
-   * };
-   *
-   * request.get(csvUrl)
-   *   .pipe(table.createWriteStream(metadata))
-   *   .on('job', (job) => {
-   *     // `job` is a Job object that can be used to check the status of the
-   *     // request.
-   *   })
-   *   .on('complete', (job) => {
-   *     // The job has completed successfully.
-   *   });
-   *
-   * //-
-   * // Load data from a JSON file.
-   * //-
-   * const fs = require('fs');
-   *
-   * fs.createReadStream('./test/testdata/testfile.json')
-   *   .pipe(table.createWriteStream('json'))
-   *   .on('job', (job) => {
-   *     // `job` is a Job object that can be used to check the status of the
-   *     // request.
-   *   })
-   *   .on('complete', (job) => {
-   *     // The job has completed successfully.
-   *   });
    */
-  createWriteStream(metadata) {
+  createWriteStream_(metadata) {
```

Comment:
Why the separation?

Reply:
We call `createWriteStream` within `createLoadJob`, however with the addition of https://github.com/googleapis/nodejs-bigquery/pull/85/files#diff-6494577af08481767e965304488d997cR1253, we introduced a race condition that could fire the jobs complete event off before the job was actually returned to the user.

We want to wait to start polling the job until the user starts, so I think this would be the best way to go in order to preserve the changes from #85.